rest_framework/apps.py

```diff
@@ -8,3 +8,4 @@ class RestFrameworkConfig(AppConfig):
     def ready(self):
         # Add System checks
         from .checks import pagination_system_check  # NOQA
+        from .checks import www_authenticate_behavior_setting_check  # NOQA
```

Comment:
Is this line necessary? In my local build I was able to trigger the new error without it; I merely copied the pattern from the line above in my PR.